scripts/scil_compute_NODDI.py

```diff
@@ -0,0 +1,185 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute NODDI [1] maps using AMICO.
+"""
+
+import argparse
+import os
+import shutil
+import tempfile
+
+import amico
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+amico.core.setup()
+
+EPILOG = """
+Reference:
+    [1] Zhang H, Schneider T, Wheeler-Kingshott CA, Alexander DC.
+        NODDI: practical in vivo neurite orientation dispersion
+        and density imaging of the human brain.
+        NeuroImage. 2012 Jul 16;61:1000-16.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, epilog=EPILOG,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    p.add_argument('in_dwi',
+                   help='DWI file acquired with a NODDI compatible protocol')
+
+    p.add_argument('--mask',
+                   help='Mask filename')
+
+    g = p.add_argument_group('Gradients / scheme')
+    g.add_argument('--bval',
+                   help='Bval filename, in FSL format')
+    g.add_argument('--bvec',
+                   help='Bvec filename, in FSL format')
+    g.add_argument('--scheme_file',
+                   help='AMICO scheme file '
+                        'can replace --bval/--bvec.')
+    g.add_argument('--bstep', type=int, nargs='+',
+                   help='List of unique bvals in your data. It prevents errors'
+                        ' when each bval is around the actual bval')
+
+    p.add_argument('--para_diff', type=float, default=1.7e-3,
+                   help='Axial diffusivity (AD) in the CC. [%(default)s]')
+    p.add_argument('--iso_diff', type=float, default=3e-3,
+                   help='Mean diffusivity (MD) in ventricles. [%(default)s]')
+    p.add_argument('--lambda1', type=float, default=2,
+                   help='First regularization parameter. [%(default)s]')
+    p.add_argument('--lambda2', type=float, default=1e-3,
+                   help='Second regularization parameter. [%(default)s]')
+
+    p.add_argument('--output_dir',
+                   help='Output directory for the NODDI results. '
+                        '[%(default)s]')
+
+    p.add_argument('--processes', type=int, default=1,
+                   help='Number of processes used to compute NODDI. Right now,'
+                        'better performance with 1. [%(default)s]')
+
```

Comment:
You should look at what is happening with the COMMIT script. We need to synchronize because the underlying mecanism is the same. 

How are you handling kernels?